### PR TITLE
fix(number-input): add displayName and __ui__ properties to improve d…

### PR DESCRIPTION
**Pull request by Itsuki54**
### Description

Added `displayName` and `__ui__` properties to the `@yamada-ui/number-input` component to improve debugging in React Developer Tools and ensure proper internal usage within Yamada UI.

### Current behavior (updates)

Currently, the `@yamada-ui/number-input` component does not have `displayName` or `__ui__` set, making it difficult to identify in React Developer Tools.

### New behavior

With this change, the `@yamada-ui/number-input` component now has both `displayName` and `__ui__` set, making it easier to identify in React Developer Tools and improving internal management within Yamada UI.

### Is this a breaking change (Yes/No):

**No**

### Additional Information

There are no breaking changes, so this update should not affect existing Yamada UI users.

### CODE REVIEW

File: .changeset/bright-toes-repeat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/number-input": minor
```
Comment (illionillion) [inline .changeset/bright-toes-repeat.md:2]: Please change to `patch`.

```suggestion
"@yamada-ui/number-input": patch
```

File: packages/components/number-input/src/number-input.tsx
Comment (illionillion) [inline packages/components/number-input/src/number-input.tsx:1]: - `NumberInputField`
- `NumberInputAddon`
- `NumberDecrementStepper`
- `NumberIncrementStepper`

Please fix it the same way for all components.

**Review by illionillion — CHANGES_REQUESTED** [review]
@Itsuki54 

Thank for PR!!

Please correct them.

File: packages/components/number-input/src/number-input.tsx
Comment (illionillion) [reply packages/components/number-input/src/number-input.tsx:1]: All other components, if any, should be modified in the same way.

**Comment by cidkumagai** [comment]
@Itsuki54

Hi, Thank for PR.
Since one changeset should be enough, please delete one of them.

**Review by illionillion — APPROVED** [review]
LGTM

**Review by cidkumagai — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**